android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -59,6 +62,27 @@
   private static final RegionResolver REGION_RESOLVER = new RegionResolver();
   private static final String SEPARATOR = "\0";
 
+  public static Comparator<PwPair> getDistanceComparator() {
```

Comment:
Please rename to newDistanceComparator (getDistanceConstructor suggests it will return the same instance each time)